www/_template/guides/tailwind-css.md

```diff
@@ -36,8 +36,8 @@ Follow our [PostCSS guide](/guides/postcss) to set up PostCSS in your Snowpack p
 module.exports = {
   plugins: [
     // ...
-    require('tailwindcss'),
-    require('autoprefixer'),
+    tailwindcss: {},
```

Comment:
I've never seen this syntax before, don't feel comfortable merging this (unless there's some docs about this being the new preferred way to do it).

Reply:
@FredKSchott absolutely, just saw it in the [TailwindCSS docs](https://tailwindcss.com/docs/installation#add-tailwind-as-a-post-css-plugin) and I see it used here, in the [postcss-load-config documentation](https://github.com/postcss/postcss-load-config#object); in this last one, they do mention that plugins can be passed in Object (with curly braces) or Array (with require) notation, so they seem to be equivalent - but unrelated to the issue with loading tailwindcss, so sorry for any confusion. 

Thanks for merging!